ballista/scheduler/src/state/execution_graph/execution_stage.rs

```diff
@@ -350,9 +349,10 @@ impl UnresolvedStage {
         )?;
 
         // Optimize join order and statistics based on new resolved statistics
-        let optimize_join = JoinSelection::new();
-        let config = SessionConfig::default();
-        let plan = optimize_join.optimize(plan, config.options())?;
+        // let optimize_join = JoinSelection::new();
+        // let config = SessionConfig::default();
+        // let plan = optimize_join.optimize(plan, config.options())?;
```

Comment:
If not needed, maybe just remove them?

Reply:
We do want to enable this again once https://github.com/apache/datafusion/issues/10978 is fixed in DataFusion. I added a comment with a link to this issue